pipeline/src/pipeline.rs

```diff
@@ -264,6 +266,7 @@ impl<T: FieldElement> Pipeline<T> {
     pub fn with_tmp_output(self, tmp_dir: &mktemp::Temp) -> Self {
         Pipeline {
             output_dir: Some(tmp_dir.to_path_buf()),
+            force_overwrite: true,
```

Comment:
yea but in the new test I'm running the pipeline twice from constant pols, which causes the witness to be generated twice. Without this the second one fails. I can also generate a new entry point just for `force_overwrite` since that doesn't exist now, but this felt easier.